orca-core/src/main/java/com/netflix/spinnaker/orca/conditions/Condition.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2019 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.orca.conditions;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.Objects;
+
+public class Condition {
```

Comment:
are we trying to go away from Lombok? I think it will generate all this boiler plate for you

Reply:
No Lombok.

Reply:
I don't actually know if we're trying to get away from lombok. I don't think we can ever actually get away from it, so I've actually started using it because it's less of a pain. 🤷‍♀️